### PR TITLE
[readme] note that brew is an unofficial installation method

### PR DESCRIPTION
**Pull request by andrei-toterman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,9 @@ Since it supports metadata for cloud-init, you can simulate a small cloud deploy
   sudo snap install multipass
   ```
 
-* On **macOS**, you can download the installer [from GitHub](https://github.com/canonical/multipass/releases) or use [Homebrew](https://github.com/Homebrew/brew):
+* On **macOS**, download the installer [from GitHub](https://github.com/canonical/multipass/releases).
+
+  Alternatively, you can use [Homebrew](https://github.com/Homebrew/brew). But please not that this is **not an officially supported** method, as it is not maintained by the Multipass team, but by the community. Multipass is available as a cask:
```
Comment (ricab) [inline README.md:25]:
```suggestion
  Alternatively, you can use [Homebrew](https://github.com/Homebrew/brew). Please note that this is **not an officially supported** method, as it is not maintained by the Multipass team, but by the community. Multipass is available as a cask:
```

**Review by ricab — DISMISSED** [review]
Looks good, but I have a 2 in 1 suggestion.

File: README.md [hunk 2]
```diff
@@ -20,7 +20,9 @@ Since it supports metadata for cloud-init, you can simulate a small cloud deploy
   sudo snap install multipass
   ```
 
-* On **macOS**, you can download the installer [from GitHub](https://github.com/canonical/multipass/releases) or use [Homebrew](https://github.com/Homebrew/brew):
+* On **macOS**, download the installer [from GitHub](https://github.com/canonical/multipass/releases).
+
+  Alternatively, you can use [Homebrew](https://github.com/Homebrew/brew). Please note that this is **not an officially supported** method, as it is not maintained by the Multipass team, but by the community. Multipass is available as a cask:
```
Comment (giuliazanchi) [inline README.md:25]:
```suggestion
  Alternatively, you can use [Homebrew](https://github.com/Homebrew/brew). Please note that this method is **not officially supported**, as it is not maintained by the Multipass team, but by the community. Multipass is available as a cask:
```

**Comment by ricab** [comment]
Should we merge or update this?

**Comment by andrei-toterman** [comment]
If I understood correctly, we're ok with mentioning brew here, so it's ready to be merged.

**Review by ricab — APPROVED** [review]
OK then!

**Pull request merged**